docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -398,6 +398,29 @@ The connector supports pushdown for a number of operations:
 * :ref:`limit-pushdown`
 * :ref:`topn-pushdown`
 
+In addition, the connector supports :ref:`aggregation-pushdown` for the
+following functions:
+
+* :func:`avg()`
+* :func:`count()`, also ``count(distinct x)``
+* :func:`max()`
+* :func:`min()`
+* :func:`sum()`
+
+Pushdown is only supported for ``DOUBLE`` type columns with the
+following functions:
```

Comment:
Some such type limitations also apply to the functions above as well but they can vary based on type mappings and what the actual data-type is in remote system.

Longer term I would prefer to move docs to a direction where we clearly set the expectation that operations will be pushed down where possible without trying to overspecify those conditions since it's complex, will never get 100% accurate due to the amount of possible combinations and will change without much visibility.

Reply:
The answer is if it doesn't work then it's unsupported. We obviously need a better way to communicate that for sure.

For a taste of complexity to know whether `sum(x)` will be pushed down:

- data type of x as Trino sees it
- the underlying data type in remote database
- any implicit casts that get added by engine (may vary based on query shape)
- the data type of the grouping columns used
- whether x is an expression or a plain column reference
- whether there are other operations that cannot be pushed down (this is unbounded list of things)